js/apps/system/_admin/aardvark/APP/react/src/views/g6graphs/useFetch.js

```diff
@@ -0,0 +1,20 @@
+/* global arangoFetch */
```

Comment:
getApiRouteForCurrentDB

Reply:
Will be solved in separate PR. Ticket: https://arangodb.atlassian.net/browse/FE-122?atlOrigin=eyJpIjoiYTA4MDA2ODBlYWZmNDI3MGEyMjc5Y2ZhODY3NTdhNzYiLCJwIjoiaiJ9